guides/common/assembly_logging-and-reporting-problems.adoc

```diff
@@ -2,7 +2,33 @@
 
 include::modules/con_logging-and-reporting-problems.adoc[]
 
-include::modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc[leveloffset=+1]
+include::modules/con_default-logging-levels.adoc[leveloffset=+1]
```

Comment:
I'd prefer to keep the defaults in the TOC.

Level offset -> 2

Reply:
I bumped the leveloffset for all component-specific procedures.